openedx/core/djangoapps/xblock/runtime/runtime.py

```diff
@@ -241,6 +241,8 @@ def service(self, block, service_name):
                 anonymous_user_id=self.anonymous_student_id,
             )
         elif service_name == "mako":
```

Comment:
When uploading transcripts in a video I am getting:

```
 WARNING 121 [openedx.core.djangoapps.xblock.runtime.runtime] [user None] [ip 172.22.0.14] runtime.py:108 - thirdparty handlers are not supported by this runtime for XBlock <class 'xblock.internal.VideoBlockWithMixins'>.
```

Reply:
That's weird @symbolist , I'm not having any trouble adding transcripts to videos. I tried each of these scenarios, and didn't get an error:

* Add transcript to new Video
* Add transcript to existing Video
* Add transcript to new Video (beta)
* Add transcript to existing Video (beta)